starfish/_display.py

```diff
@@ -246,9 +253,12 @@ def display(
             return viewer
 
         coords = coords[mask, :]
-        sizes = sizes[mask]
+        sizes = sizes[mask, :]
```

Comment:
shouldn't this be equivalent to `sizes = sizes[mask]`?

Reply:
In this regime, sizes is a (spots, 5) matrix, just like coords.  Can you point me at `napari` tests? I'd like to ideally add some to test various edge cases (2d, 3d, pseudo-3d where `z` is a singleton)

Reply:
I mean those two lines behave exactly the same, assuming `sizes` is of the right shape; just don't see the need for the explicit `, :`. There are no napari tests for high-level things like that other than creating a script and looking at it yourself so that's just what I would do.